test/Faker/Provider/mn_MN/PersonTest.php

```diff
@@ -9,12 +9,12 @@ final class PersonTest extends TestCase
 {
     public function testName()
     {
-        self::assertMatchesRegularExpression('/^[А-Я]{1}\.[\w\W]+$/u', $this->faker->name);
+        self::assertMatchesRegularExpression('/^[\p{Cyrillic}]\.[\w\W]+$/u', $this->faker->name);
```

Comment:
This looks like an unrelated change to me. Can we revert it, please?

Reply:
This change is intended - see PR description.